pkg/tenant/tenants.go

```diff
@@ -94,7 +94,7 @@ func (s *Server) TenantDetail(c *gin.Context) {
 		return
 	}
 
-	// Get the specified tenant from the database and return a 500 response
+	// Get the specified tenant from the database and return a 404 response
```

Comment:
Comments were updated to match the correct status codes that will be returned.

Reply:
Thanks!